### PR TITLE
tools: ceph-release-notes: escape asterisks not for inline emphasis

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
see also https://github.com/ceph/ceph/pull/16198/files#diff-d31d1b025d45818c51583cfa5f76c9c9R597

**Comment by tchaikov** [comment]
retest this please.

File: src/script/ceph-release-notes [hunk 1]
```diff
@@ -196,6 +196,8 @@ def make_release_notes(gh, repo, ref, plaintext, verbose, strict, use_tags):
                 title = split_component(title, gh, number)
 
             title = title.strip(' \t\n\r\f\v\.\,\;\:\-\=')
+            # escape asterisks, which is used by markdown for inline emphasis
```
Comment (smithfarm) [inline src/script/ceph-release-notes]: Did you mean reStructuredText instead of markdown?

Comment (tchaikov) [inline src/script/ceph-release-notes]: ahh, right! lemme fix the comment.

**Pull request merged**